manifests/s/SurgeSynth/Surge/1.9.0/SurgeSynth.Surge.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SurgeSynth.Surge
+PackageVersion: 1.9.0
+PackageLocale: en-US
+Publisher: Surge Synth Team
+PackageName: Surge

```

```suggestion
PackageName: SST Surge
```
Sadly it needs to pass validation once more because it needs to match ARP :c